frontend/dashboard/components/RepoList/ActionLinks.module.css

```diff
@@ -5,14 +5,21 @@
 }
 
 .giteaButton {
-  margin-right: 0.7rem;
+  margin-right: 0.45rem;
 }
 
 .giteaIcon {
   font-size: 2rem;
```

Comment:
Can we change this to a token too? 😄 Goes for entire file

Reply:
Fixed 😊